src/transformers/models/bark/modeling_bark.py

```diff
@@ -373,7 +373,7 @@ def _upad_input(self, query_layer, key_layer, value_layer, attention_mask, query
 
 
 BARK_ATTENTION_CLASSES = {
-    "default": BarkSelfAttention,
+    "eager": BarkSelfAttention,
     "flash_attention_2": BarkSelfFlashAttention2,
 }
```

Comment:
we should document in the configuraiton file which modes are available and what it means